source/braille.py

```diff
@@ -1583,6 +1588,7 @@ def terminate(self):
 			self._cursorBlinkTimer.Stop()
 			self._cursorBlinkTimer = None
 		config.post_configProfileSwitch.unregister(self.handlePostConfigProfileSwitch)
+		brailleViewer.destroyBrailleViewer()

```

Comment:
Note that Braille is initialized before and terminated after the gui. When gui is terminated before braille, does this call actually make sense?

I'm also worried that resetting the configuration to currently saved config will terminate the braille viewer and not initialize it again. As the braille viewer does not talk to the braille handler apart from at initialization time, I think it is safe to leave this out.